src/Agent.Worker/Handlers/Handler.cs

```diff
@@ -26,6 +26,8 @@ public interface IHandler : IAgentService
 
     public abstract class Handler : AgentService
     {
+        private const int _environmentVariableMaximumSize = 32766;
```

Comment:
add some comments to explain the limitation.

Reply:
done.